www/js/main.js

```diff
@@ -360,28 +350,13 @@ angular.module('emission.main', ['emission.main.recent', 'emission.main.diary',
     $ionicPopover.fromTemplateUrl('templates/control/main-collect-settings.html', {
         scope: $scope
     }).then(function(popover) {
-        $scope.collectSettingsPopup = popover;
```

Comment:
This seems like a wrong change from the merge. This should be `collectSettingsPopup` not `settingsPopup`